lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster.hs

```diff
@@ -1031,10 +1031,11 @@ clusterEraFromEnv =
         _ -> die $ var ++ ": unknown era"
     withDefault = fromMaybe maxBound
 
-localClusterConfigsFromEnv :: IO (Tagged "cluster-configs" String)
+localClusterConfigsFromEnv :: IO (Tagged "cluster-configs" FilePath)
 localClusterConfigsFromEnv = lookupEnvNonEmpty "LOCAL_CLUSTER_CONFIGS"
     <&> Tagged @"cluster-configs"
-        . fromMaybe "../local-cluster/test/data/cluster-configs"
+    . fromMaybe
+        (".." </> "local-cluster" </> "test" </> "data" </> "cluster-configs")
```

Comment:
This change is now irrelevant to fixing the GH windows unit tests, but it shouldn't hurt. It might be good for people running `cabal test` on windows after checking the repo out.